# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/gdb/x32.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- gdb-11.1/gdb/nat/amd64-linux-siginfo.c.vanilla	2021-10-18 23:55:12.793981350 +0200
+++ gdb-11.1/gdb/nat/amd64-linux-siginfo.c	2021-10-19 00:08:20.581949939 +0200
@@ -254,6 +254,11 @@
     {
       unsigned int _addr;
       unsigned int _addr_lsb;
+      struct
+      {
+        unsigned int _lower;
+        unsigned int _upper;
+      } si_addr_bnd;
     } _sigfault;
 
     /* SIGPOLL */
